app/celery_app.py

```
@@ -0,0 +1,52 @@
+# Redis remains as the result backend
+REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:6379/0")
+
+# Create SQS client using EC2 instance role
+sqs = boto3.client("sqs", region_name=AWS_REGION)
+
+# Optionally create the queue if it doesn't exist
+try:
+    response = sqs.create_queue(QueueName=SQS_QUEUE_NAME)
+    queue_url = response["QueueUrl"]
+    print(f"SQS Queue URL: {queue_url}")
+except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)